README.md

```diff
@@ -13,7 +13,10 @@
     $ cd autofdo
     $ mkdir build
     $ cd build 
-    $ cmake -DCMAKE_C_COMPILER=clang-10 -DCMAKE_CXX_COMPILER=clang++-10 -DCMAKE_BUILD_TYPE=Release ../
+	$ # To build LLVM Propeller support
+    $ cmake -DENABLE_TOOL=LLVM -DCMAKE_C_COMPILER=clang-10 -DCMAKE_CXX_COMPILER=clang++-10 -DCMAKE_BUILD_TYPE=Release ../
+	$ # Or to build tools for GCC
+	$ cmake -DENABLE_TOOL=GCOV -DCMAKE_C_COMPILER=clang-10 -DCMAKE_CXX_COMPILER=clang++-10 -DCMAKE_BUILD_TYPE=Release ../
```

Comment:
Please make the indentation consistent. Also add a new line between the LLVM and GCC version invocations.

A couple of other fixes for this file:
* Update the target platform to Ubuntu 22.04
* Update the prerequisites to include libzstd-dev instead of zstd
* Change "Build autofdo tools using clang-10" to "Build autofdo tools"
* Change "To build LLVM Propeller support" to "Build LLVM tools for AutoFDO and Propeller"
* Add the contents of @dhoekwater [comment](https://github.com/google/autofdo/pull/200#issuecomment-2226223676) about cmake support as a separate section for 20.04 

Did you check that both cmake invocations work with gcc?

Reply:
Done.

Reply:
> Did you check that both cmake invocations work with gcc?

Yes, tested building with Ubuntu 22.04 LTS GCC/G++ (11.4.0), both builds (GCOV/LLVM) succeeded with #210 (already merged).